### DISCUSSION TITLE
Remove Date from Card and sort

### DISCUSSION THREAD

**Discussion by abwalker417** [firstof9/lovelace-grocy-meal-plan-card#33] [Ideas]
Is there a way to hide the date on the card. 

Also is there a way to sort so the card displays the meal order (Breakfast, Lunch, Dinner) I have the sections built like that

**Comment by firstof9** [comment]
> Is there a way to hide the date on the card.

Not at this time.


> Also is there a way to sort so the card displays the meal order (Breakfast, Lunch, Dinner) I have the sections built like that

Likely something that could be added, but as it is now, they're not sorted.
